app/views/courses/_basic_details_tab.html.erb

```diff
@@ -109,7 +109,7 @@
           Full or part time
         </dt>
         <dd class="govuk-summary-list__value" data-qa="course__study_mode">
-          <%= course.study_mode.humanize %>
+          <%= course.study_mode&.humanize %>
```

Comment:
I thought we wanted to move humanize into the decorator?

Reply:
I tried to move it to the decorator as a bonus...
But a number of things was using the raw value, and I can't seems to find a way to access the undecorated value...